build/variables.yml

```diff
@@ -69,9 +72,16 @@
   # Flutter version.
   FlutterVersion: '3.22.1'
 
+  # Docker version.
```

Comment:
I think it would be worth explaining why we need this now. It's not clear why we would need docker in a Flutter project.

Reply:
done